plugin/http_client.py

```diff
@@ -162,6 +162,15 @@ def do_request_from_buffer():
         open_scratch_buffer(response, vim_ft)
 
 
+def write_content(content, buffer):
```

Comment:
I'd suggest `write_to_buffer` as a clearer name. Also `contents` for the variable name.